--- rsnapshot.conf.default.in	2006-05-14 11:01:23.000000000 +1000
+++ rsnapshot.conf.default.in.redhat	2006-05-14 13:50:13.000000000 +1000
@@ -45,7 +45,7 @@
 #
 # See the README file or the man page for more details.
 #
-#@CMD_CP@
+@CMD_CP@
 
 # uncomment this to use the rm program instead of the built-in perl routine.
 #
@@ -68,7 +68,7 @@
 # If you have an older version of "du", you may also want to check the
 # "du_args" parameter below.
 #
-#@CMD_DU@
+@CMD_DU@
 
 # Uncomment this to specify the path to rsnapshot-diff.
 #
@@ -117,14 +117,14 @@
 # If you enable this, data will be written to the file you specify. The
 # amount of data written is controlled by the "loglevel" parameter.
 #
-#logfile	/var/log/rsnapshot
+logfile	/var/log/rsnapshot
 
 # If enabled, rsnapshot will write a lockfile to prevent two instances
 # from running simultaneously (and messing up the snapshot_root).
 # If you enable this, make sure the lockfile directory is not world
 # writable. Otherwise anyone can prevent the program from running.
 #
-#lockfile	/var/run/rsnapshot.pid
+lockfile	/var/run/rsnapshot.pid
 
 # Default rsync args. All rsync commands have at least these options set.
 #
