--- rsnapshot.conf.default.in	2005-01-22 19:07:05.000000000 -0800
+++ rsnapshot.conf.default.in.REDHAT	2005-01-22 19:26:14.000000000 -0800
@@ -34,7 +34,7 @@
 #
 # See the README file or the man page for more details.
 #
-#@CMD_CP@
+@CMD_CP@
 
 # uncomment this to use the rm program instead of the built-in perl routine.
 @CMD_RM@
@@ -49,7 +49,7 @@
 @CMD_LOGGER@
 
 # Uncomment this to specify a path to "du" for disk usage checks.
-#@CMD_DU@
+@CMD_DU@
 
 #########################################
 #           BACKUP INTERVALS            #
@@ -127,7 +127,7 @@
 # If you enable this, make sure the lockfile directory is not world
 # writable. Otherwise anyone can prevent the program from running.
 #
-#lockfile	/var/run/rsnapshot.pid
+lockfile	/var/run/rsnapshot.pid
 
 ###############################
 ### BACKUP POINTS / SCRIPTS ###
