--- rsnapshot.conf.default.in	2005-01-24 15:01:00.000000000 -0800
+++ rsnapshot.conf.default.in.rpm	2005-01-24 15:02:53.000000000 -0800
@@ -40,7 +40,7 @@
 #
 # See the README file or the man page for more details.
 #
-#@CMD_CP@
+@CMD_CP@
 
 # uncomment this to use the rm program instead of the built-in perl routine.
 @CMD_RM@
@@ -55,7 +55,7 @@
 @CMD_LOGGER@
 
 # Uncomment this to specify a path to "du" for disk usage checks.
-#@CMD_DU@
+@CMD_DU@
 
 #########################################
 #           BACKUP INTERVALS            #
@@ -133,7 +133,7 @@
 # If you enable this, make sure the lockfile directory is not world
 # writable. Otherwise anyone can prevent the program from running.
 #
-#lockfile	/var/run/rsnapshot.pid
+lockfile	/var/run/rsnapshot.pid
 
 ###############################
 ### BACKUP POINTS / SCRIPTS ###
