--- rsnapshot.conf	Wed Nov 26 20:25:51 2003
+++ rsnapshot.conf-RPM	Wed Nov 26 20:31:01 2003
@@ -24,7 +24,7 @@
 #
 cmd_rsync	/usr/bin/rsync
 cmd_ssh		/usr/bin/ssh
-#cmd_cp		/bin/cp
+cmd_cp		/bin/cp
 
 # BACKUP INTERVALS
 # (must be unique and in ascending order, i.e. hourly, daily, weekly, etc.)
@@ -40,7 +40,7 @@
 #rsync_long_args      --delete --numeric-ids --devices
 
 # make sure the lockfile directory is not world writable!
-#lockfile	/var/lock/subsys/rsnapshot
+lockfile	/var/lock/subsys/rsnapshot
 
 # don't span filesystems within a backup point
 #one_fs		0
