--- rsnapshot.conf.default	2005-01-29 03:16:08.000000000 -0800
+++ rsnapshot.conf.default.RPM	2005-01-29 03:17:00.000000000 -0800
@@ -40,7 +40,7 @@
 #
 # See the README file or the man page for more details.
 #
-#cmd_cp		/bin/cp
+cmd_cp		/bin/cp
 
 # uncomment this to use the rm program instead of the built-in perl routine.
 cmd_rm		/bin/rm
@@ -55,7 +55,7 @@
 cmd_logger	/usr/bin/logger
 
 # Uncomment this to specify a path to "du" for disk usage checks.
-#cmd_du		/usr/bin/du
+cmd_du		/usr/bin/du
 
 #########################################
 #           BACKUP INTERVALS            #
@@ -94,7 +94,7 @@
 
 # If you enable this, data will be written to the file you specify. The
 # amount of data written is controlled by the "loglevel" parameter.
-#logfile	/var/log/rsnapshot
+logfile		/var/log/rsnapshot
 
 # The include and exclude parameters, if enabled, simply get passed directly
 # to rsync. If you have multiple include/exclude patterns, put each one on a
