--- rsnapshot.conf.default.in	Fri Dec 26 17:19:18 2003
+++ rsnapshot.conf.default.in.RPM	Fri Dec 26 17:24:05 2003
@@ -28,7 +28,7 @@
 #
 # See the README file or the man page for more details.
 #
-#@CMD_CP@
+@CMD_CP@
 
 # rsync must be enabled for anything to work.
 @CMD_RSYNC@
@@ -67,7 +67,7 @@
 # If you enable the lockfile, make sure the lockfile directory is
 # not world writable. Otherwise anyone can prevent the program from running.
 #
-#lockfile	/var/lock/subsys/rsnapshot
+lockfile	/var/lock/subsys/rsnapshot
 
 # If this is enabled, rsync won't span filesystem partitions within a
 # backup point. This essentially passes the -x option to rsync.
