--- rsnapshot.conf.default.in	2005-01-29 22:56:58.000000000 -0800
+++ rsnapshot.conf.default.in.redhat	2005-01-29 22:57:44.000000000 -0800
@@ -55,7 +55,7 @@
 @CMD_LOGGER@
 
 # Uncomment this to specify a path to "du" for disk usage checks.
-#@CMD_DU@
+@CMD_DU@
 
 #########################################
 #           BACKUP INTERVALS            #
@@ -94,7 +94,7 @@
 
 # If you enable this, data will be written to the file you specify. The
 # amount of data written is controlled by the "loglevel" parameter.
-#logfile	/var/log/rsnapshot
+logfile		/var/log/rsnapshot
 
 # The include and exclude parameters, if enabled, simply get passed directly
 # to rsync. If you have multiple include/exclude patterns, put each one on a
@@ -138,7 +138,7 @@
 # If you enable this, make sure the lockfile directory is not world
 # writable. Otherwise anyone can prevent the program from running.
 #
-#lockfile	/var/run/rsnapshot.pid
+lockfile	/var/run/rsnapshot.pid
 
 ###############################
 ### BACKUP POINTS / SCRIPTS ###
