--- rsnapshot.conf.default.in	2003-12-19 18:56:53.000000000 -0800
+++ rsnapshot.conf.default.in.RPM	2003-12-19 19:06:38.000000000 -0800
@@ -24,7 +24,7 @@
 #
 @CMD_RSYNC@
 #@CMD_SSH@
-#cmd_cp		/bin/cp
+cmd_cp		/bin/cp
 
 # BACKUP INTERVALS
 # (must be unique and in ascending order, i.e. hourly, daily, weekly, etc.)
@@ -43,7 +43,7 @@
 #ssh_args			-p 22
 
 # make sure the lockfile directory is not world writable!
-#lockfile	/var/lock/subsys/rsnapshot
+lockfile	/var/lock/subsys/rsnapshot
 
 # don't span filesystems within a backup point
 #one_fs		0
