--- rsnapshot.conf.default.in	Mon May 17 00:02:47 2004
+++ rsnapshot.conf.default.in.RPM	Mon May 17 00:03:21 2004
@@ -34,7 +34,7 @@
 #
 # See the README file or the man page for more details.
 #
-#@CMD_CP@
+@CMD_CP@
 
 # uncomment this to use the rm program instead of the built-in perl routine
 #@CMD_RM@
@@ -122,7 +122,7 @@
 # If you enable the lockfile, make sure the lockfile directory is
 # not world writable. Otherwise anyone can prevent the program from running.
 #
-#lockfile	/var/run/rsnapshot
+lockfile	/var/lock/subsys/rsnapshot
 
 ###############################
 ### BACKUP POINTS / SCRIPTS ###
