From: Guillaume Delacour <gui@iroqwa.org>
Subject: Fix path call pg_dumpall
Bug-Debian: http:/bugs.debian.org/500082
Forwarded: not-needed
Last-Update: 2013-07-31

--- a/utils/backup_pgsql.sh
+++ b/utils/backup_pgsql.sh
@@ -3,18 +3,16 @@
 ##############################################################################
 # backup_pgsql.sh
 #
+# Debian modifications by Antony Gelberg <antony.gelberg@wayforth.com>
+#
 # by Nathan Rosenquist <nathan@rsnapshot.org>
 # http://www.rsnapshot.org/
 #
 # This is a simple shell script to backup a PostgreSQL database with rsnapshot.
 #
 # The assumption is that this will be invoked from rsnapshot. Also, since it
-# will run unattended, the user that runs rsnapshot (probably root) should have
-# a .pgpass file in their home directory that contains the password for the
-# postgres user. For example:
-#
-# /root/.pgpass (chmod 0600)
-#   *:*:*:postgres:thepassword
+# will run unattended, the user that runs rsnapshot (probably root) must su
+# to Debian's postgres user.
 #
 # This script simply needs to dump a file into the current working directory.
 # rsnapshot handles everything else.
@@ -25,7 +23,12 @@
 umask 0077
 
 # backup the database
-/usr/local/pgsql/bin/pg_dumpall -Upostgres > pg_dumpall.sql
+su -l -c 'pg_dumpall > ~/pg_dumpall.sql' postgres
 
 # make the backup readable only by root
-/bin/chmod 600 pg_dumpall.sql
+chown root.root ~postgres/pg_dumpall.sql
+chmod 600 ~postgres/pg_dumpall.sql
+
+# Move the backup to current working directory
+mv ~postgres/pg_dumpall.sql `pwd`
+
